src/lms-harmonizer/tests_integration_sql/helpers/sql_scripts.py

```diff
@@ -13,9 +13,9 @@
 EXEC sys.sp_executesql N'CREATE SCHEMA lms';
         """
 
-CREATE_SCHEMA_EDFILMS = """
-IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = N'edfilms')
-EXEC sys.sp_executesql N'CREATE SCHEMA edfilms';
+CREATE_SCHEMA_LMSX = """
+IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = N'LMSX')
+EXEC sys.sp_executesql N'CREATE SCHEMA LMSX';
```

Comment:
I wonder if we could instead install the `lms-ds-loader` as a dev dependency and run it on an empty directory (no data upload) to create these tables?

That is an idea for a future improvement. Not worth pursuing today.